ihatemoney/forms.py

```diff
@@ -51,6 +52,8 @@ class CalculatorStringField(StringField):
     in StringField
     """
 
+    widget = NumberInput()
```

Comment:
Can you explicit why this is needed? :-)

Reply:
It's needed to use input type=number instead of text.... This help display the corresponding keyboard on mobile... 


But you mean in the code?

Reply:
The thing is that numbers and ` + - * /` operators are actually accepted, and by using the NumberInput I believe it would not be possible to enter operations in this field, isn't it?